02-oop-inheritance/5/scripts/movie.js

```diff
@@ -0,0 +1,64 @@
+class EventEmitter {
+    constructor() {
+        this.events = new Map(); 
+    }
+
+    on (eventName, listener) {
```

Comment:
Remove extra space between `on` and `(` or add that extra space after each function name declared in this file.